lapis/job_io/htcondor.py

```diff
@@ -31,36 +32,52 @@ def htcondor_job_reader(
         "DiskUsage_RAW": 1.024 / 1024 / 1024,
     },
 ):
-    htcondor_reader = csv.DictReader(iterable, delimiter=" ", quotechar="'")
+    input_file_type = iterable.name.split(".")[-1]
+    if input_file_type == "json":
+        htcondor_reader = json.load(iterable)
+    elif input_file_type == "csv":
+        htcondor_reader = csv.DictReader(iterable, delimiter=" ", quotechar="'")
+    else:
+        print("Invalid input file type {}. Job input file can not be read.".format(
+            input_file_type))
```

Comment:
Two comments here:

1) You could go the pythonic way here. Try loading the json and if an exception occurs, go for csv. This prevents having issues determining the file type based on different cases.
2) Please don't simply print error messages. You could either `assert` that you got the correct type or you could use the *implementation* logger to properly log the error.